mne/preprocessing/ica.py

```diff
@@ -2302,9 +2306,9 @@ def _plot_corrmap(data, subjs, indices, ch_type, ica, label, show, outlines,
 
 
 @verbose
-def corrmap(icas, template, threshold="auto", label=None,
-            ch_type="eeg", plot=True, show=True, verbose=None, outlines='head',
-            layout=None, sensors=True, contours=6, cmap='RdBu_r'):
+def corrmap(icas, template, threshold="auto", label=None, ch_type="eeg",
+            plot=True, show=True, verbose=None, outlines='head', layout=None,
+            sensors=True, contours=6, cmap='RdBu_r'):
```

Comment:
@jona-sassenhagen what @jaeilepp says is that when you have positive valued maps we take by default "reds_r" as color map. This is the case when we compute topos from gradiometers by taking the norm of a given sensor pair. So cmap should be None and resolved in a channel type dependent manner.